src/pages/AssignServerRoles.js

```diff
@@ -52,6 +53,9 @@ class AssignServerRoles extends BaseWizardPage {
   constructor(props) {
     super(props);
 
+    this.IS_INSTALLL = this.props.mode ? false : true;
```

Comment:
typo here, too (should be IS_INSTALL). Actually this check is confusing because I thought that `this.props.mode` is some kind of string that contains either `addserver` or something else. If it is really a boolean, then can we rename it to something more meaningful like `this.props.addMode` or even `this.props.installMode`? That would also remove the need for this extra `IS_INSTALL` variable since the name would be more meaningful.

Reply:
you could explicitly check for 'addserver' here, or check explicitly against undefined.... the logic makes sense, but I can see where at a glance it makes this.props.mode look like a boolean

Reply:
thanks for clarify, will update